api/wasm/cpp/proxy_wasm_api.h

```diff
@@ -315,8 +315,14 @@ class RootContext : public ContextBase {
   RootContext* asRoot() override { return this; }
   Context* asContext() override { return nullptr; }
 
+  // Can be used to validate the configuration (e.g. in the control plane). Returns false if the
+  // configuration is invalid.
+  virtual bool onValidateConfiguration(std::unique_ptr<WasmData> /* configuration */) {
```

Comment:
Nit: not sure if `on` prefix is justified here.

Reply:
changed to validateConfiguration